mlflow/ml-package-versions.yml

```diff
@@ -16,7 +16,6 @@ sklearn:
     maximum: "0.24.1"
     requirements: ["matplotlib"]
     run: |
-      pytest tests/sklearn/test_sklearn_training_session.py --large
```

Comment:
`tests/sklearn/test_sklearn_training_session.py` has been renamed to `tests/autologging/test_training_session.py` which is executed in `dev/run-python-flavor-tests.sh`.